.ci/riscv-tests.sh

```diff
@@ -7,20 +7,23 @@ check_platform
 mkdir -p toolchain
 
 # GNU Toolchain for RISC-V
-GCC_VER=12.2.0-1
-XPACK_REPO=https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack/releases/download
+GCC_VER=2023.01.04-nightly
+TOOLCHAIN_REPO=https://github.com/riscv-collab/riscv-gnu-toolchain/releases
+
+# Install RISCOF
+pip3 install git+https://github.com/riscv/riscof.git
```

Comment:
Use the command below instead.
```shell
python3 -m pip install git+https://github.com/riscv/riscv-ctg
```

It can eliminate some potential packaging problems.

Reply:
Shall I rewite the command as following ?
```
python3 -m pip install git+https://github.com/riscv/riscof
```

Reply:
Yes, please go ahead.